shared/js/smart-screen/f_t_e_wizard.js

```diff
@@ -0,0 +1,192 @@
+/* global SimpleKeyNavigation, KeyEvent */
+/* exports FTEWizard */
+
+'use strict';
+
+(function(exports) {
+  const MAX_TRANSITIONEND_LATENCY = 4000;
+
+  function FTEWizard(name) {
+    this._name = name;
+    this._running = false;
+
+    this._launched = !!localStorage.getItem(name + '.fteskip');
+    this._finished = this._launched;
+  }
+
+  Object.defineProperty(FTEWizard.prototype, 'running', {
+    get: function() {
+      return this._running;
+    }
+  });
+
+  Object.defineProperty(FTEWizard.prototype, 'finished', {
+    get: function() {
+      return this._finished;
```

Comment:
If we return ```this._launched``` here, we can remove the ```this._finished``` property. We can still leave it like this if you'd like, I'm OK with either way.